src/piton/labelers/omop_inpatient_admissions.py

```diff
@@ -0,0 +1,161 @@
+"""Labeling functions for OMOP data based on lab values."""
+from __future__ import annotations
+
+import datetime
+from typing import List
+
+from .. import Event, Patient
+from ..extension import datasets as extension_datasets
+from .core import (
+    Label,
+    Labeler,
+    LabelType,
+    TimeHorizon,
+    TimeHorizonEventLabeler,
+)
+from .omop import (
+    WithinVisitLabeler,
+    _get_all_children,
+    get_death_concepts,
+    get_inpatient_admission_concepts,
+    get_inpatient_admission_discharge_times,
+    get_inpatient_admission_events,
+)
+
+
+class DummyAdmissionDischargeLabeler(Labeler):
```

Comment:
Do we need to call this Dummy?

Reply:
Ethan suggested that as a name to make it explicit its not a "true" labeler @Lalaland